hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -18,7 +18,7 @@
   dv_doc:             "../doc/dv",
   hw_checklist:       "../doc/checklist",
   sw_checklist:       "/sw/device/lib/dif/dif_usbdev",
-  version:            "1.0.0",
+  version:            "2.0.0",
```

Comment:
Increasing major version due to multiple SW-visible backwards-incompatible changes

Reply:
Thanks. There are a couple of differences in the register API that apply to normal operation.